broadcast/src/linked/mod.rs

```diff
@@ -0,0 +1,616 @@
+//! Ordered, reliable broadcast across multiple, dynamic participants.
```

Comment:
While I suggested `dynamic` here, I think that is actually the wrong word (implies that participants can come/go during an epoch).

Let's use `Ordered, reliable broadcast across reconfigurable participants.`